packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.js

```diff
@@ -258,6 +258,32 @@ class MoreInfoBox extends React.Component {
 				)
 		}
 	}
+	renderMoreInfoBox() {
+		const contentDescription = []
+
+		if (this.props.type === 'ObojoboDraft.Chunks.List') {
+			contentDescription.push({
+				name: 'spacing',
+				description: 'List',
+				type: 'select',
+				values: [
+					{
+						value: 'compact',
+						description: 'compact'
```

Comment:
The descriptions of these should be uppercased, like `Compact` instead of `compact`